README.md

```diff
@@ -32,37 +32,6 @@ git clone https://github.com/yearn-integrations/yearn-api.git
 
 ```
 cd yearn-api
-cp .sample.env.local .env
-```
-
-- Start the docker container
-
-```
-docker-compose up
-```
-
-- The API should now be running locally on your dev machine
-
-### Optional - Configure environment variables
-
-- .sample.env.local and .sample.env.dev contain sample API keys for various services (Infura/Etherscan). This is done to enable developers to get up to speed quickly. If you are planning on developing APIs that extensively utilize these keys please consider generating new API keys :)
-- Update your .env file to use your own custom web3, archive node, and graph endpoints
-- Update your .env file to use your own custom AWS Credentials (if using .sample.env.dev and not .sample.sample.env.local)
-
-## Usage Instructions
-
-### Use "Offline Mode" for local development and testing
-
-- Run the command `docker-compose up` to test API endpoints locally
-- You can reach the API under localhost:3000
-  - use localhost:3000/local or localhost:3000/dev, depending on your environment
-  - If you want to change the local port, change the "ports" entry under "serverless" in the docker-compose.yml
-- The dirs config, services and utils are mounted into the running container, so you code changes will become available instantly on the running instance
-  - If you want to change that, remove the "volumes" entries under "serverless" in the docker-compose.yml
-
-## Stages
-
-- Currently four stages are available
```

Comment:
what is intention behind removing this README section?

Reply:
I totally believe this is a mistake. Shoud I fix it and push again?

Reply:
solved